### DISCUSSION TITLE
Disqus

### DISCUSSION THREAD

**Discussion by joshisanonymous** [Automattic/wordpress-activitypub#195] [Q&A] [answered]
For commenting interactions between the actual blog and the fediverse, does this require using the default WordPress commenting system? For example, if I use Disqus for comments on my WordPress blog, does that mean it won't be possible for fediverse users to comment on posts from the fediverse?

**Comment by timnolte** [comment, marked as answer]
Yes, this plugin is meant to use the native WordPress comments. The whole reason for this it to keep the content on your site. Using Disqus means your site doesn't have any of that content. I really wouldn't recommend using Disqus any more.
